LongestPalindrome.java

```diff
@@ -0,0 +1,58 @@
+//Time Complexity = O(n)
+//Space.Complexity = O(n)
```

Comment:
space would be contamt since you are allowed only lower and upper case letter only which will be O(52) = O(1)